app-frontend/src/app/components/filterPane/filterPane.html

```diff
@@ -16,10 +16,30 @@
       <div class="filter-item">
         <div id="year_slider"></div>
       </div>
-      <rzslider rz-slider-model="$ctrl.yearFilters.minModel"
-                rz-slider-high="$ctrl.yearFilters.maxModel"
-                rz-slider-options="$ctrl.yearFilters.options"
-      ></rzslider>
+      <div class="btn-group">
+        <button type="button"
+                class="btn btn-toggle btn-small"
+                ng-class="{'active': $ctrl.yearFilterMode === 'range'}"
+                ng-click="$ctrl.setYearFilterMode('range')">
```

Comment:
Currently I'm getting strange behavior if I click "Year Range" twice without switching to "Single year" in between. If I just keep clicking "Year Range" then it extends the lower bound of the range by one year each time I click. Even stranger, however, if I click "Year Range" and then drag the year slider to cover 2016-2017, and then click "Year Range" again, it ends up with the max year at 2010 and the min year at 2011:
![weirdyears](https://cloud.githubusercontent.com/assets/447977/22343630/d8ea0946-e3c6-11e6-8aac-6bccdeec9021.png)

I wonder if we should cache the year filter mode on the controller; then we could run the switch logic only when it changes to avoid these kinds of edge cases.